packages/composite-checkout/src/components/billing-fields.js

```diff
@@ -507,51 +448,65 @@ function BillingFormSummary( { paymentData } ) {
 	return (
 		<GridRow gap="4%" columnWidths="48% 48%">
 			<div>
-				<BillingSummaryParagraphs>
-					{ paymentData.billingName } <br />
-					{ paymentData.billingAddress } <br />
-					{ paymentData.billingCity }, { paymentData.billingState } <br />
-					{ paymentData.billingZipCode || paymentData.billingPostalCode },{' '}
-					{ paymentData.billingCountry }
-				</BillingSummaryParagraphs>
-
-				<BillingSummaryParagraphs>
-					{ paymentData.billingPhoneNumber }
+				<BillingSummaryDetails>
+					<BillingSummaryLine>{ billingData.name } </BillingSummaryLine>
+					<BillingSummaryLine>{ billingData.address } </BillingSummaryLine>
+					<BillingSummaryLine>
+						{ billingData.city }, { billingData.state || billingData.province }{' '}
```

Comment:
Can we get rid of the `{' '}` at the end of this line either by using another template string or just leveraging the `BillingSummaryLine`? (It's also possible that token might have been added by Prettier if you had a space at the end of the line accidentally.)

Reply:
Yeah, that's Prettier. I'll convert it to a literal.